tools/cli/admin_db_clean_command_test.go

```diff
@@ -24,21 +24,46 @@ package cli
 
 import (
 	"flag"
+	"fmt"
 	"os"
 	"testing"
 
+	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/urfave/cli/v2"
+
+	"github.com/uber/cadence/common/persistence"
+	"github.com/uber/cadence/common/reconciliation/invariant"
 )
 
-func TestAdminDBClean_errorCases(t *testing.T) {
+func TestAdminDBClean_noFixExecution(t *testing.T) {
 	tests := []struct {
 		name           string
 		setupContext   func(app *cli.App) *cli.Context
 		inputFileData  string // Simulate the content of the input file
 		expectedOutput string
 		expectedError  string
 	}{
+		{
+			name: "SuccessCase_emptyResult",
+			setupContext: func(app *cli.App) *cli.Context {
+				set := flag.NewFlagSet("test", 0)
+				// Define flags
+				set.String(FlagScanType, "", "scan type flag")
+				set.String(FlagInputFile, "", "Input file flag")
+				// Use a StringSliceFlag to simulate multiple collection values
+				set.Var(cli.NewStringSlice("CollectionHistory", "CollectionDomain"), FlagInvariantCollection, "invariant collection flag")
+
+				// Set actual values for the flags
+				_ = set.Set(FlagScanType, "ConcreteExecutionType")
```

Comment:
nit: check for errors in tests also. If this fails, the test will fail with an unrelated error, and it will be harder to track the issue.

My suggestion - use 
```
require.NoError(t, set.Set(FlagScanType, "ConcreteExecutionType"))
```